content/rdi/installation/install-k8s.md

```diff
@@ -14,10 +14,36 @@ Throughout the document, the snippets make use of the Kubernetes `kubectl` tool.
 
 ## Prerequisites
 
-- An existing Redis Enterprise cluster version >= 6.2.
-- [RedisGears](https://redis.com/modules/redis-gears/) >= 1.2.5 installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install.
+- An existing Redis Enterprise cluster version >= 6.2.18
+- [RedisGears](https://redis.com/modules/redis-gears/) {{<param rdi_redis_gears_current_version>}} installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install. 
```

Comment:
remove redundant space after the period.

Reply:
done